content/tutorials/webrtc/basics/en/index.html

```diff
@@ -184,57 +186,57 @@ <h2 id="toc-mediastream">MediaStream (aka getUserMedia)</h2>
 
-<h2 id="toc-signaling">Signaling: session control, network and media information</h2>
+<h2 id="toc-signaling">Signaling: Session control, network, and media information</h2>
 
-<p>WebRTC uses RTCPeerConnection to communicate streaming data between browsers (aka peers), but also needs a mechanism to coordinate communication and to send control messages, a process known as signaling. Signaling methods and protocols are <em>not</em> specified by WebRTC: signaling is not part of the RTCPeerConnection API.</p>
+  <p>WebRTC uses <code>RTCPeerConnection</code> to communicate streaming data between browsers (also known as peers), but also needs a mechanism to coordinate communication and to send control messages, a process known as signaling. Signaling methods and protocols are <em>not</em> specified by WebRTC. Signaling is not part of the <code>RTCPeerConnection</code> API.</p>
 
-<p>Instead, WebRTC app developers can choose whatever messaging protocol they prefer, such as SIP or XMPP, and any appropriate duplex (two-way) communication channel. The <a href="https://appr.tc" title="apprtc WebRTC example">appr.tc</a> example uses XHR and the Channel API as the signaling mechanism. The <a href="https://codelabs.developers.google.com/codelabs/webrtc-web/#0" title="WebRTC codelab">codelab</a> we built uses <a href="http://Socket.io" title="Socket.io website">Socket.io</a> running on a <a href="http://nodejs.org/" title="Node website">Node server</a>.</p>
+<p>Instead, WebRTC app developers can choose whatever messaging protocol they prefer, such as SIP or XMPP, and any appropriate duplex (two-way) communication channel. The <a href="https://appr.tc" title="apprtc WebRTC example">appr.tc</a> example uses XHR and the Channel API as the signaling mechanism. The <a href="https://codelabs.developers.google.com/codelabs/webrtc-web/#0" title="WebRTC codelab">codelab</a> uses <a href="https://socket.io" title="Socket.io website">Socket.io</a> running on a <a href="https://nodejs.org/" title="Node website">Node server</a>.</p>
 
 <p>Signaling is used to exchange three types of information:</p>
 <ul>
-  <li>Session control messages: to initialize or close communication and report errors.</li>
-  <li>Network configuration: to the outside world, what's my computer's IP address and port?</li>
-  <li>Media capabilities: what codecs and resolutions can be handled by my browser and the browser it wants to communicate with?</li>
+  <li>Session control messages, which initialize or close communication and report errors.</li>
```

Comment:
These three aren't quite right in the new version (i.e. the meaning is changed).

I'd be inclined to change these back — can't think of a way to make them use the same grammatical format.

Reply:
Will change back